common/traits/hfe_leader_trait.txt

```diff
@@ -90,7 +96,7 @@ hfe_ruler_trait_warriror_honor = {
 		# opinion effect
 		hfe_planet_jobs_war_militarist_produces_mult = 0.1
 		# TODO
-		#rivalries_influence_produces_mult = 0.5
```

Comment:
Dont remove this line, one i still want to make some better effect instead of vanilla one.

Reply:
Okay, done